src/helpers/data/load-meeting-data.js

```diff
@@ -487,16 +487,16 @@ function flattenDays(data) {
       indexes_to_remove.push(index);
       meeting.day.forEach(day => {
         meetings_to_add.push({
+          ...meeting,
           day: day,
           slug: meeting.slug + '-' + day,
-          ...meeting,
         });
       });
     }
   });
 
   indexes_to_remove.forEach(index => {
-    data = data.splice(index, 1);
+    data.splice(index, 1);
```

Comment:
and `data = data.splice(index, 1);` was adding the newly created objects _with_ the old one instead of mutating in place